### ISSUE TITLE
Evaluate our support for submitting beyond [.01, .99] range for metaculus questions

### ISSUE THREAD

**Issue by djthorne** [oughtinc/ergo#288]
TLDR: We should either really figure out what is going on with the low and high submission fields or simply enforce a [.01, .99] window. 

metaculus.py currently is trying to use larger submission ranges when self.low_open and self.high_open are not False (get passed "tail" for `low` or `high` in the question's `possibility` object). We also test this with continuous_linear_open_question in metaculus.py. Here is the possibilities object for that questions:

`{'low': 'tail',
 'high': 'tail',
 'type': 'continuous',
 'scale': {'max': 1000000, 'min': 1, 'deriv_ratio': 1},
 'format': 'num'}`

however, if you pass a submission object with a `low` or `high` field >.99, it will not except it
That violates the assumptions of metaculus.py. 

The current state is that I've enforced the [.01, .99] window, but kept the logic such that we can easily adjust once we understand what is going on. Alternatively if we just want to stop supporting this feature we can simplify the code considerably
